setup.py

```diff
@@ -31,6 +31,7 @@
         'requests-cache>=0.4.13',
         'tabulate>=0.8.3',
         'pycryptodome>=3.8.2',
+        'jsbeautifier==1.11.0'
```

Comment:
Doing this apparently has the potential to break this for some AUR and brew users. Perhaps move this to `extras_require` (add to existing, and one of its own)?

Reply:
how tf does it have such a potential tho?
its not our problem if aur is fucked up, is it?

Reply:
in the first place we dont the aur package is an unofficial one...

Reply:
It's unofficial? Strange that vn-ki would comment about it (see: https://github.com/anime-dl/anime-downloader/pull/443#issuecomment-662447368)

Reply:
ok fine, you win, ill make it an optional dependency